### ISSUE TITLE
[ADD] Option to recover/restore/import existing Vault on first run

### ISSUE THREAD

**Issue by RowboTony** [vultisig/vultisig-windows#452]
**Is your feature request related to a problem? Please describe.**
I don't want to create a new Vault on a fresh install/first run of the Windows app. 

**Describe the solution you'd like**
Provided option for [advanced] or [import/restore/recover]

**Describe alternatives you've considered**
I appreciate the walk-through screen on furst run, and the clarity of on-boarding a new user, but being forced to create a throw-away Vault on new install in order to get to the (+Add New Vault) / (Import Existing Vault) buttons is not neccessary for every user.

Can these be added to the main screen on first launch? Or somewhere advanced users can find without creating a new Vault?

![vultisig-windows-import](https://github.com/user-attachments/assets/8441dcd6-c5d6-424b-af1d-080906e35928)

**Comment by johnnyluo** [comment]
@RowboTony , you have to have at least 1 vault ot use vultisig , so you either create one , or import one that has been created somewhere else , if you don't have any vault , mostly feature in Vultisig can't be accessable , thus I don't think we will change this

**Comment by RowboTony** [comment]
Thanks. I installed v1.0.3 in a Windows sandbox and was presented **only** with these screens upon first run.

![vultisig-splash](https://github.com/user-attachments/assets/b10c5f88-5539-4093-95ca-aca551a1a148)
![vultisig-create](https://github.com/user-attachments/assets/7880c20e-986e-4257-8b92-c42d717e6864)

When I compiled the latest repo from source, also v1.0.3 (build 3) I did receive the option I was asking about. The compiled dev version didn't force me to create a new vault. I don't know if it detects other vaults on my system or what. Either way, this issue is resolved. Thanks!

![vultisig-compile-import-ok](https://github.com/user-attachments/assets/07bc90cf-f7d8-4e74-aab4-90af8c26346f)
